app/views/analysis/index.html.erb

```diff
@@ -0,0 +1,600 @@
+            <span class="fa fa-plus"></span> Race
+            </button>
+              <div id="race_chosen" class="collapse">
+                <input type="checkbox" name="race" value="White"> White<br>
+                <input type="checkbox" name="race" value="Black or African American"> Black or African American<br>
+                <input type="checkbox" name="race" value="American Indian or Alaskan Native (specify tribe)"> American Indian or Alaskan Native<br>
+                <input type="checkbox" name="race" value="Asian Indian"> Asian Indian<br>
+                <input type="checkbox" name="race" value="Chinese"> Chinese<br>
+                <input type="checkbox" name="race" value="Filipino"> Filipino<br>
+                <input type="checkbox" name="race" value="Japanese"> Japanese<br>
+                <input type="checkbox" name="race" value="Korean"> Korean<br>
+                <input type="checkbox" name="race" value="Vietnamese"> Vietnamese<br>
+                <input type="checkbox" name="race" value="Native Hawaiian"> Native Hawaiian<br>
+                <input type="checkbox" name="race" value="Guamanian or Chamorro"> Guamanian or Chamorro<br>
+                <input type="checkbox" name="race" value="Samoan"> Samoan<br>
+              </div>
+            </div>
+          </div>
+          <div class="row js-filter" id="other_criteria" style="display:none">
+            <div align="left">
+              <button type="button" class="btn btn-link" data-toggle="collapse" id="specify_criteria" data-target="#other_options">
+            <span class="fa fa-plus"></span> Other Options
+            </button>
+              <div id="other_options" class="collapse">
+                <input type="checkbox" name="specified_criteria" value="served_armed_forces"> Only show Armed Forces service members<br>
+                <input type="checkbox" name="specified_criteria" value="tobacco_cause"> Only show tobacco-related deaths<br>
+              </div>
+            </div>
+          </div>
+          <br>
+        </div>
+        <br>
+      </div>
+      <div class="col-md-8 offset-md-1">
+        Chart Display
+        <div class="row">
+          <div align="left">
+            <button type="button" onclick="common_deaths_age()" class="btn btn-primary">Most Common Causes Of Death</button>&nbsp;
+          </div>
+          <div align="center">
+            <button type="button" onclick="display_causes_map()" class="btn btn-primary">Display Map</button>&nbsp;
+          </div>
+          <div class="right">
+            <button type="button" id="save_chart" class="btn btn-primary" align="right">Save Chart Image</button>&nbsp;
+          </div>
+        </div>
+        <div class="row mt-3">
+          <p id="demo"></p>
+          <div id="chart" align="center"></div>
+          <div id="map"></div>
+        </div>
+      </div>
+    </div>
+    <br>
+
+  </form>
+  <p><tt id="results"></tt></p>
```

Comment:
I don't think `tt` is a thing anymore in HTML5 [https://www.w3schools.com/tags/tag_tt.asp](https://www.w3schools.com/tags/tag_tt.asp) could this be replaced with something else?